tests/integration/integration_test.go

```diff
@@ -43,6 +43,8 @@ func abortIfFailed(t *testing.T) {
 type policyTestScenario struct {
 	// WantErrors is the error message we expect to see in the command's output.
 	WantErrors []string
+	// Whether the error messages are advisory, and don't actually fail the command.
```

Comment:
Nit: Is "command" the best term to use here? Would "operation" be better? IMHO, it seems like a more natural fit. What do you think?